tests/test_resolver.py

```diff
@@ -10,9 +10,33 @@
 
 # This environment var must be set before the usd imports.
 os.environ["TF_DEBUG"] = "OPENASSETIO_RESOLVER"
-from pxr import Plug, Usd
+from pxr import Plug, Usd, Ar
 
 
+# Assume OpenAssetIO is configured as the custom primary resolver for
+# all tests. If you're wondering where this is configured, it may
+# just be set via the `PXR_PLUGINPATH_NAME` environment variable.
+
+# Verify OpenAssetIO configured as the AR resolver.
+@pytest.fixture(autouse=True)
+def openassetio_configured():
+    pr = Plug.Registry()
+    plugin = pr.GetPluginWithName("usdOpenAssetIOResolver")
+
+    assert (
+        plugin is not None
+    ), "OpenAssetIO is not configured as a resolver. Set `PXR_PLUGINPATH_NAME` to `plugInfo.json`"
+
+
+# Set the default resolver search path to a subdirectory
+def build_search_path_context(path_relative_from_file):
+    script_dir = os.path.realpath(os.path.dirname(__file__))
+    full_path = os.path.join(script_dir, path_relative_from_file)
+    return Ar.DefaultResolverContext([path_relative_from_file])
```

Comment:
[fix] Should this be `full_path`?

```suggestion
    return Ar.DefaultResolverContext([full_path])
```

Reply:
Ah, yes, another left over symptom of all my search path hacking. Annoyingly (or not, depending on your outlook,) USD will append the working directory if it detects the path is not absolute, so this continues to "work". But as you mentioned before, this would fall over if you ran the tests from an unexpected working directory.

Reply:
https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/13/commits/ef46ffb6ca0fda46223aa4fb1e6aff175b7d41de